sdk/search/search-documents/src/searchIndexClient.ts

```diff
@@ -458,6 +458,8 @@ export class SearchIndexClient {
         utils.publicSynonymMapToGeneratedSynonymMap(synonymMap),
         updatedOptions
       );
+      console.log(`Create _response: ${(result as any)._response.headers}`);
```

Comment:
@xirzec @EmmaZhu This line is the breaking change. Instead of accessing it as `result._response`, I am accessing it as `(result as any)._response`.

Reply:
I think this is fine, individual service packages can handle exposing the typing if they want (like how storage documented it in their api report), but it will still work with trivial changes to consumer code.